client/verta/verta/client.py

```diff
@@ -180,15 +180,16 @@ def __init__(self, host=None, port=None, email=None, dev_key=None,
                 response = _utils.make_request("GET",
                                                "{}://{}/api/v1/modeldb/project/verifyConnection".format(conn.scheme, conn.socket),
                                                conn)
-            except requests.ConnectionError:
-                six.raise_from(requests.ConnectionError("connection failed; please check `host` and `port`"),
-                               None)
+            except requests.ConnectionError as err:
+                conn_error_msg = "connection failed; please check `host` and `port`; original error: \n{}".format(str(err))
+                six.raise_from(requests.ConnectionError(conn_error_msg), None)
 
             def is_unauthorized(response): return response.status_code == 401
 
             if is_unauthorized(response):
-                auth_error_msg = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`"
-                six.raise_from(requests.HTTPError(auth_error_msg), None)
+                # response.reason was "Unauthorized"
+                new_reason = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`; {}".format(response.reason)
```

Comment:
Again, not totally sure how nice this will look, but it's what I'm imaging would look cleaner.

```suggestion
                new_reason = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`\n\n{}".format(response.reason)
```

Reply:
```python3
Traceback (most recent call last):
  File "../connection_err.py", line 2, in <module>
    Client()
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/client.py", line 194, in __init__
    _utils.raise_for_http_error(response)
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/_internal_utils/_utils.py", line 534, in raise_for_http_error
    six.raise_from(e, None)  # use default reason
  File "/Users/nhatpham/Documents/modeldb/client/verta/verta/external/six/__init__.py", line 737, in raise_from
    raise value
requests.exceptions.HTTPError: 401 Client Error: authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`

Unauthorized for url: ... at 2020-11-09 23:44:17.315000 UTC
```
I think you only one new line, no?

Reply:
We do only need one line, but I personally think it's nicer to have a gap between our error message and the original.
Do you have any thoughts on that? 😄

Reply:
Nope, it looks nicer this way actually. Just making sure this is what you want.